packages/moonstone/Picker/Picker.js

```diff
@@ -187,14 +197,23 @@ const PickerBase = kind({
 				validateRange(value, 0, max, 'Picker', '"value"', 'min', 'max index');
 			}
 			return clamp(0, max, value);
+		},
+		voiceLabel: ({children, 'data-webos-voice-labels-ext': voiceLabelsExt}) => {
+			let voiceLabel;
+			if (voiceLabelsExt) {
+				voiceLabel = voiceLabelsExt;
+			} else {
+				voiceLabel = children.map((child) => ((typeof child === 'number' || typeof child === 'string') ? child : ''));
```

Comment:
I might be good to verify that `children` does exist and is an array (or array-like object) before using.  Even better would be to use `React.Children.map()`

Reply:
I did add array check condition of children and change to React.Children.map(). Thanks~